ethereum/spec/src/main/resources/tech/pegasys/teku/spec/config/configs/chiado.yaml

```diff
@@ -156,10 +156,15 @@ BELLATRIX_FORK_EPOCH: 180 # Mon Oct 10 2022 14:00:00 GMT+0000
 # Capella
 CAPELLA_FORK_VERSION: 0x0300006f
 CAPELLA_FORK_EPOCH: 244224 # Wed May 24 2023 13:12:00 GMT+0000
+# Deneb
+DENEB_FORK_VERSION: 0x0400006f
+DENEB_FORK_EPOCH: 516608 # Wed Jan 31 2024 18:15:40 GMT+0000
 
 INACTIVITY_SCORE_BIAS: 4
 # 2**4 (= 16)
 INACTIVITY_SCORE_RECOVERY_RATE: 16
+# [New in Deneb:EIP7514] 2**3 (= 8)
+MAX_PER_EPOCH_ACTIVATION_CHURN_LIMIT: 8
```

Comment:
I'm not sure on this. It's set it to [2 for gnosis](https://github.com/gnosischain/specs/blob/master/consensus/config/gnosis.yaml#L77-L78) but I don't see a line in chiado config.

Reply:
oh is it? I couldnt find it and it was needed, so i copied from mainnet... if theres a gnosis one we can copy that, but ideally it should be in their actual config :/ same as fork_choice settings which were missing